deployment/bicep/shared/vm-nic.bicep

```diff
@@ -3,10 +3,10 @@ param location string = resourceGroup().location
 param subnetId string
 param privateIPAddress string =  '10.0.0.4'
 
-param vmName string
+param nicName string
```

Comment:
I got a chuckle out of `nic(k)Name`

I'm sorry, I had to. Bad humor. Just ignore me. =P